File: CS/WebSolution.Module/Classes.cs
using System;

using DevExpress.Xpo;

using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Validation;

namespace WebSolution.Module {
    [DefaultClassOptions]
    public class MyPerson : Person {
        public MyPerson(Session session) : base(session) { }
        [Association("MyPerson-MyTasks")]
        public XPCollection<MyTask> MyTasks {
            get {
                return GetCollection<MyTask>("MyTasks");
            }
        }
    }
    public class MyTask : Task {
        public MyTask(Session session) : base(session) { }
        private MyPerson _myPerson;
        [Association("MyPerson-MyTasks")]
        public MyPerson MyPerson {
            get {
                return _myPerson;
            }
            set {
                SetPropertyValue("MyPerson", ref _myPerson, value);
            }
        }
    }
}

File: VB/WebSolution.Module/Classes.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.Xpo

Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Validation

Namespace WebSolution.Module
	<DefaultClassOptions> _
	Public Class MyPerson
		Inherits Person
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		<Association("MyPerson-MyTasks")> _
		Public ReadOnly Property MyTasks() As XPCollection(Of MyTask)
			Get
				Return GetCollection(Of MyTask)("MyTasks")
			End Get
		End Property
	End Class
	Public Class MyTask
		Inherits Task
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Private _myPerson As MyPerson
		<Association("MyPerson-MyTasks")> _
		Public Property MyPerson() As MyPerson
			Get
				Return _myPerson
			End Get
			Set(ByVal value As MyPerson)
				SetPropertyValue("MyPerson", _myPerson, value)
			End Set
		End Property
	End Class
End Namespace
